.eslintrc

```diff
@@ -15,16 +9,22 @@
         "ignore": ["^([a-zA-Z@]+[-\\.]?)+"]
       }
     ],
-    "lodash/import-scope": [2, "method"],
     // Rules that conflict with Prettier.
     // To verify conflicting rules, run:
     // $ npm run prettier-conflict-check
-    "react/jsx-curly-spacing": 0,
-    "react/jsx-equals-spacing": 0,
-    "react/jsx-indent": 0,
-    "react/jsx-indent-props": 0,
-    "react/jsx-tag-spacing": 0,
-    "react/jsx-wrap-multilines": 0
+    "@typescript-eslint/no-empty-function": "off",
+    "@typescript-eslint/camelcase": "off",
+    "@typescript-eslint/no-use-before-define": "off",
+    "@typescript-eslint/class-name-casing": "off",
+    // TODO: re-enable these rules
```

Comment:
Remove comment since it's been registered in issue https://github.com/vtex/styleguide/issues/986

Reply:
ok!